packages/system/gpu-operator/charts/gpu-operator/templates/nodefeaturerules.yaml

```
@@ -0,0 +1,107 @@
+        sev.es.enabled: "true"
+      matchFeatures:
+      - feature: cpu.security
+        matchExpressions:
+          sev.es.enabled:
+            op: IsTrue
+    - name: SEV system capacities
+      extendedResources:
+        sev_asids: '@cpu.security.sev.asids'
+        sev_es: '@cpu.security.sev.encrypted_state_ids'
+      matchFeatures:
+      - feature: cpu.security
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)